sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeEntitiesWithResolutions.md

```diff
@@ -0,0 +1,172 @@
+# Resolve entities to standard formats with NER resolutions
+
+This sample demonstrates how to resolve entities in a document by using the named entity recognition (NER) feature of the Azure Cognitive Service for Language. NER resolutions provide standard, predictable formats for common types and concepts (such as dates, quantities, and dimensions) to help you retrieve and process information from documents more efficiently. To learn more about NER resolutions, see [here][NER_Resolutions].
+
+## Create a `TextAnalyticsClient`
+
+To create a new `TextAnalyticsClient`, you will need the service endpoint and credentials of your Language resource. To authenticate, you can use the [`DefaultAzureCredential`][DefaultAzureCredential], which combines credentials commonly used to authenticate when deployed on Azure, with credentials used to authenticate in a development environment. In this sample, however, you will use an `AzureKeyCredential`, which you can create simply with an API key.
+
+```C# Snippet:CreateTextAnalyticsClient
+Uri endpoint = new("<endpoint>");
+AzureKeyCredential credential = new("<apiKey>");
+TextAnalyticsClient client = new(endpoint, credential);
+```
+
+The values of the `endpoint` and `apiKey` variables can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
+
+## Use NER resolutions
+
+In this sample, we will use the following documents to try to illustrate several different types of resolutions.
+
+```C# Snippet:Sample4_RecognizeEntitiesBatchWithResolutionsAsync_GetDocuments
+string documentA = "The dog is 14 inches tall and weighs 20 lbs. It is 5 years old.";
+string documentB = "This is the first aircraft of its kind. It can fly at over 1,300 mph and carry 65-80 passengers.";
+string documentC = "The apartment (840 sqft with 2 bedrooms) costs 2,000 USD per month and will be available on 11/01/2022.";
+string documentD = "Mix 1 cup of sugar. Bake for 60 minutes in an oven preheated to 350 degrees F.";
+string documentE = "They retrieved 200 terabytes of data from 10/24/2022 to 10/28/2022.";
+
+List<TextDocumentInput> batchedDocuments = new() {
+    new TextDocumentInput("1", documentA),
+    new TextDocumentInput("2", documentB),
+    new TextDocumentInput("3", documentC),
+    new TextDocumentInput("4", documentD),
+    new TextDocumentInput("5", documentE),
+};
```

Comment:
Opinion: I'd number the documents to correspond when their indexes instead e.g. "1" -> "document1", etc. Easier to see which docs are for which indexes.

Reply:
I've been cleaning up all the TA samples little by little and will make this change as part of a separate PR in the future because I think it's a good idea. At least for now all the samples follow this pattern consistently.